frontend/src/app/testResults/__snapshots__/TestResultPrintModal.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TestResultPrintModal should render a list of tests 1`] = `<div />`;
```

Comment:
This doesn't look right

Reply:
No, it doesn't 🙂 It took a little more finagling to get this straightened out but it is fixed in 091e148.